README.md

```diff
@@ -23,7 +23,7 @@ Note that the OS platform used by the maintainers is macOS but the tools and set
 
 * Clone the project and submodules:
 ```
-$ git clone --recurse-submodules https://github.com/automattic/gutenberg-mobile
+$ git clone --recurse-submodules git@github.com:wordpress-mobile/gutenberg-mobile.git
```

Comment:
Good eye @ajitbohra , thanks for submitting this PR!

Can we change it back to `https://` though? That should be more accessible to people since they won't need to have SSH keys added to their GitHub account, or even an account at all.

Reply:
done :+1: